studies/templates/studies/_all_json_and_csv_data.html

```diff
@@ -71,22 +67,28 @@ <h1>All Responses</h1>
             <div class="col-sm-3">
                 {% if active == 'all' %}
                     <a id='download-all-data-json' class ='btn btn-primary' href="{% url 'exp:study-responses-download-json' pk=study.id %}">
-                        Download All Responses as JSON</span>
+                        Download all {{ n_responses }} responses as JSON</span>
                     </a>
                     <a id='download-all-data-csv' class='hidden btn btn-primary' href="{% url 'exp:study-responses-download-csv' pk=study.id %}">
-                        Download All Responses as CSV</span>
+                        Download all {{ n_responses }} responses as CSV</span>
                     </a>
                 {% endif %}
                 {% if active == 'demographics' %}
                     <a id='download-all-data-json' class ='btn btn-primary' href="{% url 'exp:study-demographics-download-json' pk=study.id %}">
-                        Download All Demographic Snapshots as JSON</span>
+                        Download all {{ n_responses }} demographic snapshots as JSON</span>
```

Comment:
There's a django template `pluralize` helper you can use, for the case when there's one snapshot or response https://docs.djangoproject.com/en/2.0/ref/templates/builtins/#pluralize

Reply:
Oh cool, thank you!